upup/models/cloudup/resources/addons/authentication.aws/k8s-1.12.yaml.template

```diff
@@ -109,6 +109,25 @@ subjects:
   name: aws-iam-authenticator
   namespace: kube-system
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: kube-system
+  name: aws-iam-authenticator
+  labels:
+    k8s-app: aws-iam-authenticator
+data:
+  config.yaml: |
+    # a unique-per-cluster identifier to prevent replay attacks
+    # (good choices are a random token or a domain name that will be unique to your cluster)
+
+    clusterID: {{ ClusterName }}
```

Comment:
Why set `clusterID` here when it's already being passed as a flag? And this appears to ignore the `ClusterID` setting in `AWSAuthenticationSpec`.

Reply:
Could you please give details to reproduce the ignorance you observed?

I have observed the failing aws-iam-authenticator from [kops update cluster](https://kops.sigs.k8s.io/getting_started/aws/#build-the-cluster), this PR can fix it.